source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -17612,9 +17612,10 @@
             {
                 'ctypes_data_type': 'ctypes.c_int',
                 'direction': 'out',
+                'grpc_streaming_type': 'I32Data',
```

Comment:
Should these changes be coming from [scrapigen](https://github.com/ni/grpc-device-scrapigen)? I didn't see these changes on that side.

Reply:
Yes, this metadata should be coming from scrapigen, the PR on scrapigen with this change is not published yet as we are making changes for some more apis. It will be out tomorrow, for now I have copied the metadata from there itself.